src/main/java/bio/terra/cloudfiletodatastore/deltalayer/model/PointCorrectionOperation.java

```diff
@@ -0,0 +1,32 @@
+package bio.terra.cloudfiletodatastore.deltalayer.model;
+
+public class PointCorrectionOperation {
+
+  String op;
```

Comment:
In addition to `AddUpdateAttribute` op, does  `Delta Layer` support `DeleteAttribute` op too? Could we make this an `enum`?